packages/fuels-test-helpers/src/lib.rs

```diff
@@ -162,6 +163,24 @@ pub async fn setup_test_client(
     (client, srv.bound_address)
 }
 
+pub fn create_storage_slot<T: Sized, D: Sized>(key: T, value: D) -> StorageSlot {
+    if ::std::mem::size_of::<T>() > Bytes32::LEN || ::std::mem::size_of::<D>() > Bytes32::LEN {
+        panic!("StorageSlot key/value is limited to 32 bytes.");
+    }
+
+    unsafe fn to_bytes32<T: Sized>(p: &T) -> Bytes32 {
```

Comment:
IIUC you're taking the raw bytes representation from memory and putting that in storage? 

I'm not sure that is necessarily compatible with how the ABI requires types to be encoded -- i.e. the endianess might be different for a start. Enum encoding and other stuff is not specified in Rust and is free to be whatever the compiler wants it to be and probably other stuff as well.

I'm not savvy with storage slots but if gets sent towards the VM then we'll have a problem. 

Otherwise, if it is just for local usage it seems like a weird way to pass data around the program :D

Reply:
If it's something purely for our internal testing then it shouldn't be exposed to the user as a feature.

Reply:
I decided to remove it since its benefits are negligible.